keps/sig-architecture/2140-receipts/README.md

```diff
@@ -0,0 +1,266 @@
+TODO: Should we rewrite these into user story format? i.e. "As a SIG tech lead, I want to graduate a KEP in a specific release"
+
+#### KEP Authors
+
+##### Proposing enhancements
+
+* Specify New KEP for Release
+* Specify Graduation phase for existing KEP
+* Link / Report k/k PRs for a given Enhancment
+* Indicate k/w PRs for a given Enhancement
+
+##### Filing an exception to Enhancements/Code Freeze
+
+* Request Exception for Enhancement Freeze
+* Request Exception for Code Freeze
+
+#### API Reviewers
+
+* Review / Approve Exception Requests that need API Review
+    * Ensure reviewer bandwidth considered
+    * Priortize areas for upcoming releases
+
+#### SIG Chair / TLs / PMS
+
+* Review Open/Proposed KEPs for my SIG
+    * Help plan reviews / dependencies
+    * Help identify major themes / release blogs
+* Review At Risk KEPS for my SIG
+    * Help Prioritize last minute reviews
+* Review Exceptions for my SIG
+    * Determine release blocking / major importance
+* Review Slipped Enhancements to support planning
+    * Determine what phase it missed
+    * Identify multiple slips / misses
+
+#### Enhancements Subteam
+
+* Pre enhancements Freeze (Enhancements Collection)
+  * Collect intents of KEP authors to graduate a KEP
+  * Ensure that those intents comply to the graduation requirements
+  * Mark enhancements "At Risk" based on compliance with graduation requirements
+
+* At Enhancements Freeze
+  * Remove enhancements that do not comply with requirements
+
+* Enhancements Freeze - Code Freeze
+  * Ensure that the k/k PRs for the KEP are populated
+  * Mark enhancements "At Risk" based on progress on implementation
+  * Approve/Deny exceptions for Enhancements Freeze
+
+* Around Code Freeze
+  * Ensure that the enlisted k/k are merged
+  * Approve/Deny exceptions for Code Freeze
+
+* At Code Freeze
+  * Remove enhancements that have not completed requirements for Code Freeze
+
+* Throughout the Cycle
+  * Prepare a manifest of what's the status of enhancements at any point of time
+
+#### Docs Subteam
+
+* Enhancements Freeze - Open Docs PR Deadline
+  * Ensure that the k/w PRs are populated in the KEPs
+  * Communicate to SIG Docs about the expected review load
+
+* Open Docs PR Deadline - Docs Complete
+  * Mark enhancements "At Risk" if the following are true:
+    *  Docs PRs are not updated
+    *  The draft PRs have not been updated
+
+### Notes/Constraints/Caveats
+
+This KEP is based on a [high level plan](https://docs.google.com/document/d/1qnfXjQCBrikbbu9F38hdzWEo5ZDBd57Cqi0wG0V6aGc/edit#) incorporating feedback collected from release team members and contribtors over several cycles.
+
+In 1.21, we'll focus on implementing much of this process manually so there will be some burdon placed on people to create the appropriate YAML structure, but ideally these changes will free up time for people on the release team to work on automating
+
+### Risks and Mitigations
+
```

Comment:
Biggest risk is non-compliance, what is the mechanism for making it happen at each stage? There will be authors/sigs that do not participate for various reasons: capacity/lack of awareness/don't want to

Another Risk is that this isn't communicated effectively: this should also probably lay out how this is change of process is going to be communicated maybe in a section above?

Reply:
I have added a few points that I think could be a mitigation measure for non-compliance. Let's discuss further on those points.

Reply:
Communication is a big one, we have had some trouble with getting everyone aware of PRR, for example.